lib/app/store.js

```diff
@@ -3,20 +3,15 @@ import Vuex from 'vuex'
 
 Vue.use(Vuex)
 
-// Recursive find files in {srcDir}/{dir.store}
 const files = require.context('@/<%= dir.store %>', true, /^\.\/(?!<%= ignorePrefix %>)[^.]+\.(<%= extensions %>)$/)
```

Comment:
BTW: Is there an "ES6" equivalent for `require.context`?

Reply:
Let's keep this enhancement in another PR :) I will work on it